woodwork/tests/schema/test_table_schema_init.py

```diff
@@ -94,7 +92,7 @@ def test_check_logical_types_errors(sample_column_names):
         'phone_number': None,
         'age': None,
     }
-    error_message = re.escape("logical_types is missing columns that are present in TableSchema: ['is_registered', 'signup_date']")
+    error_message = re.escape("logical_types is missing columns that are present in TableSchema: ['boolean', 'categorical', 'datetime_with_NaT', 'double', 'double_with_nan', 'integer', 'is_registered', 'nullable_integer', 'signup_date']")
```

Comment:
Either add the new column names to the `bad_logical_types_keys` dict or break this message into multiple lines as it is really long.